.prettierignore

```diff
@@ -0,0 +1,3 @@
+*.jpg
```

Comment:
maybe add `.jpeg` too?

Reply:
good idea. I've only fixed "the errors" but adding it right now